src/custom/constants/lists.ts

```diff
@@ -8,7 +8,9 @@ export type NetworkLists = {
   [chain in ChainId]: string[]
 }
 
-const COW_DAO_LIST = 'token-list.cow.eth'
+const COW_DAO_LIST = 'https://files.cow.fi/tokens/CowSwap.json'
+const COW_COINGECKO_LIST = 'https://files.cow.fi/tokens/CowSwap-coingecko.json'
```

Comment:
When I try to open this file I'm seeing this 
![Screenshot from 2022-10-18 15-00-29](https://user-images.githubusercontent.com/34926005/196436744-7fd36034-5b35-46d8-85dd-88cd4b28bf64.png)

Reply:
Hmm, me too, but I don't know how it works.
https://github.com/cowprotocol/cowswap/pull/1179 - I see, that COW_DAO_LIST works as usual. Maybe he added COW_COINGECKO_LIST for the future?

Reply:
Oh I think it should be https://files.cow.fi/tokens/CoinGecko.json

Reply:
shouldn't we fix this?